server/public/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
Hmm, I think we do, otherwise I've got error while trying to start app 
Node screams about missing favicon `https://github.com/xsolve-pl/xsolve-feat/blob/master/server/bootstrap.js#L17`